### PR TITLE
Add a pdf parser and serializer

### PR DESCRIPTION
**Pull request by qijieye**

### CODE REVIEW

**Review by lucymeachem-p2p — APPROVED** [review]

**Review by lucymeachem-p2p — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -57,18 +57,22 @@
 		"@typescript-eslint/eslint-plugin": "5.48.1",
 		"@typescript-eslint/parser": "5.48.1",
 		"eslint": "^8.31.0",
-		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20221229",
+		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20231004",
 		"eslint-plugin-simple-import-sort": "^8.0.0",
 		"formdata-polyfill": "^4.0.10",
```
Comment (fredrikbryntesson) [inline package.json:62]: Dont mix cleanup and fixes with your actual functionality changes. Linting should be a separate pull request.

**Review by fredrikbryntesson — CHANGES_REQUESTED** [review]

**Review by fredrikbryntesson — APPROVED** [review]
